packages/debug/src/browser/editor/debug-editor-service.ts

```diff
@@ -112,10 +115,17 @@ export class DebugEditorService {
             model.toggleBreakpoint();
         }
     }
+    toggleContextBreakpoint(): void {
+        const { model } = this;
+        if (model) {
+            model.toggleContextBreakpoint();
+        }
+    }
+
     setBreakpointEnabled(enabled: boolean): void {
```

Comment:
it looks fragile, if someone calls it programmatically when the last context position will be enabled, not a breakpoint for the current line